File: ASP.NET Core/GraphQL_API/Models/Dvdlanguage.cs
using System;
using System.Collections.Generic;

namespace GraphQL_API.Models
{
    public partial class Dvdlanguage
    {
        public long DvdId { get; set; }
        public long LanguageId { get; set; }
        public long AudioFormatId { get; set; }
        public long DvdLanguageId { get; set; }

        public virtual AudioFormat AudioFormat { get; set; }
        public virtual Dvd Dvd { get; set; }
        public virtual Language Language { get; set; }
    }
}

File: Express.js/models/DVDLanguage.js
/* jshint indent: 2 */

module.exports = function(sequelize, DataTypes) {
  return sequelize.define('DVDLanguage', {
    dvd_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      references: {
        model: 'DVD',
        key: 'dvd_id'
      }
    },
    language_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      references: {
        model: 'Language',
        key: 'language_id'
      }
    },
    audio_format_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      references: {
        model: 'AudioFormat',
        key: 'audio_id'
      }
    },
    dvd_language_id: {
      type: DataTypes.INTEGER,
      allowNull: true,
      primaryKey: true
    }
  }, {
    tableName: 'DVDLanguage'
  });
};
